app/actions/configurations.py

```diff
@@ -11,31 +9,5 @@ class AuthenticateConfig(ActionConfiguration):
     refresh_token: str = "string"
 
 
-class PullObservationsHeader(ActionConfiguration):
-    Authorization: str
-
-
 class PullObservationsConfig(ActionConfiguration):
```

Comment:
```suggestion
class PullObservationsConfig(PullActionConfiguration):
```

Reply:
we should also make `AuthenticateConfig` to inherit from `AuthActionConfiguration`